packages/bigquery/src/table.js

```diff
@@ -1064,12 +871,21 @@ Table.prototype.import = function(source, metadata, callback) {
  *
  * @param {object|object[]} rows - The rows to insert into the table.
  * @param {object=} options - Configuration object.
+ * @param {boolean} options.autoCreate - Automatically create the table if it
```

Comment:
Maybe we should leave a note that this can take longer than 2 minutes to complete.